c2corg_api/tests/views/test_route.py

```diff
@@ -57,6 +57,9 @@ def test_post_non_whitelisted_attribute(self):
         }
         self.post_non_whitelisted_attribute(body)
 
+    def test_post_missing_content_type(self):
```

Comment:
Since this test is the same for all document types, can't we move it to the parent class BaseTestRest? Or maybe it would not be executed for each doc type then?